x/attribute/types/msgs.go

```diff
@@ -36,7 +36,10 @@ func (msg MsgAddAttributeRequest) ValidateBasic() error {
 	if _, err := sdk.AccAddressFromBech32(msg.Owner); err != nil {
 		return err
 	}
-	a := NewAttribute(msg.Name, msg.Account, msg.AttributeType, msg.Value, msg.ExpirationDate)
+	if len(msg.ConcreteType) > 200 {
+		return fmt.Errorf("concrete_type exceeds maximum length of 200 characters")
+	}
```

Comment:
You don't need this check since the attribute validation (called a couple lines below) will do that.